header.go

```diff
@@ -3,7 +3,7 @@ package openflow
 import (
 	"io"
 	"math/rand"
-
+	"fmt"
```

Comment:
It would be nice to keep the imports grouped by type: https://github.com/golang/go/wiki/CodeReviewComments#imports. Please, you [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) to make it automatically.